packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -71,6 +64,10 @@
                     {{ $t('buttonText') }}
                 </f-button>
             </form>
+
+            <terms-and-conditions
+                v-if="!isLoggedIn"
+                :class="$style['c-checkout-termsAndConditions']" />
```

Comment:
Could this style be inside the `TermsAndConditions` component instead?

Reply:
🤦‍♀️  yup

Reply:
Ahh just remember why I changed it. It's so I didn't have to create a width variable in both `Checkout` and `T&Cs`. Is it worth having the two variables to remove styling T&Cs twice?